clutter/agent.txt

```diff
@@ -0,0 +1 @@
+alchemicalu@Micahs-MBP
```

Comment:
I don't think this file should be under version control.

Reply:
I've deleted the unnecessary sub-directory that address this and the follow issues / changes needed @lucksus